src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.skia.cs

```diff
@@ -315,6 +291,37 @@ private static IDisposable InnerCreateLayer(UIElement owner, LayoutState state)
 			return disposables;
 		}
 
+		private static Rect CreateImageLayer(Compositor compositor, CompositeDisposable disposables, double adjustedStrokeThicknessOffset, Rect adjustedArea, CompositionSpriteShape backgroundShape, Rect backgroundArea, ImageBrush imgBackground)
+		{
+			imgBackground.Subscribe(imageData =>
+			{
+
+				if (imageData.Error is null)
+				{
+					var surfaceBrush = compositor.CreateSurfaceBrush(imageData.Value);
+
+					var sourceImageSize = new Size(imageData.Value.Image.Width, imageData.Value.Image.Height);
+
+					// We reduce the adjustedArea again so that the image is inside the border (like in Windows)
+					var imageArea = adjustedArea.DeflateBy(new Thickness((float)adjustedStrokeThicknessOffset));
+
+					backgroundArea = imgBackground.GetArrangedImageRect(sourceImageSize, imageArea);
```

Comment:
Codacy found an issue: [Introduce a new variable instead of reusing the parameter 'backgroundArea'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932478)